src/Microsoft.ApplicationInsights/Extensibility/ISerializationWriter.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// The interface for defining writers capable of serializing data into various formats.    
+    /// </summary>
+    public interface ISerializationWriter
```

Comment:
I assume we may want to support more methods in the future, did we currently scope the available methods down to the list agreed upon in some extensibility discussions? I had a question on long, List vs. Array and JSON serializer mapping, but those won't need to be answered if we simply expose the minimum in here. Is so, can/should we limit it even more, or it's already too few methods for the scenario?